lms/djangoapps/courseware/views/index.py

```diff
@@ -388,9 +388,15 @@ def _create_courseware_context(self):
             if self.section.position and self.section.has_children:
                 display_items = self.section.get_display_items()
                 if display_items:
-                    courseware_context['sequence_title'] = display_items[self.section.position - 1] \
-                        .display_name_with_default
-
+                    try:
+                        courseware_context['sequence_title'] = display_items[self.section.position - 1] \
+                            .display_name_with_default
+                    except IndexError:
+                        log.info("Course section {} with position {} and total section display items: {}".format(
```

Comment:
@adampalay i have attached the screenshot. If this error occurs, we will see something like this:
<img width="1225" alt="screen shot 2017-04-20 at 11 20 29 am" src="https://cloud.githubusercontent.com/assets/13939335/25216625/d25a617e-25bc-11e7-8e68-f89cfe793920.png">

What's your thoughts on it now?